pkg/generated/BUILD

```diff
@@ -5,6 +5,24 @@ load(
     "go_library",
 )
 
+genrule(
+    name = "bindata",
+    srcs = [
+        "//translations:all-srcs",
+    ],
+    outs = ["bindata.go"],
```

Comment:
verified by doing `bazel build pkg/generated:bindata`, then copying the generated file out, running `gofmt -w -s` on it, and diffing.